src/expipe_plugin_cinpla/tools/trackunitmulticomparison.py

```diff
@@ -0,0 +1,284 @@
+import numpy as np
+import networkx as nx
+import yaml
+from .trackunitcomparison import TrackingSession
+from expipe_plugin_cinpla.data_loader import get_data_path, get_channel_groups, load_spiketrains
+from .track_units_tools import plot_template
+import matplotlib.pylab as plt
+from tqdm import tqdm
+import uuid
+from matplotlib import gridspec
+from collections import defaultdict
+from pathlib import Path
+import datetime
+
+class TrackMultipleSessions:
```

Comment:
It would be super cool to make an additional Tab in the GUI to do this and visualize it